my-sms/app/views/notes/_form_new.html.haml

```diff
@@ -1,11 +1,12 @@
-= form_for([notable, Note.new]) do |f|
-    %div.field
-        =f.label :title
-        %br
-        =f.text_field :title
-    %div.field
-        =f.label :content
-        %br
-        =f.text_field :content
+= form_for(notable) do |notable_f|
+    =notable_f.fields_for 'notes_attributes', notable.notes.build, index: nil do |f|
```

Comment:
I've always used `new` instead of `build` and was wondering whether there was a difference...
turns out [as of rails 3.2.13](https://stackoverflow.com/a/12541282) they do the [same thing](https://github.com/rails/rails/blob/main/activerecord/lib/active_record/relation.rb#L78) (for reference, the plus repo uses rails 3.2.22.5)

Reply:
Ah good to know. I thought there was some sort of automatic behaviour `build` had when used in the form builder but guess not.